### PR TITLE
Mes ms 4783 catc1e fail journey

### PR DESCRIPTION
**Pull request by JCarterDVSA**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: test/e2e/features/cat-c1e/11-fail_journey_with multiple_faults.feature [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    And I start the test for "Mr Denny Carlson"
+    And the candidate completes the declaration page
+    And the candidate confirms their declaration
+    Then I should see the "Declaration - Denny Carlson" page
+    And the candidate enters a new email address
+    And I proceed to the car
+    Then I should see the "Denny Carlson" page
+    And I complete the waiting room to car page with the following vehicle checks
+      | show_me_1  | show_me_2  |
+      | true      | true      |
```
Comment (jono-basford-bjss) [inline test/e2e/features/cat-c1e/11-fail_journey_with multiple_faults.feature:15]: Fix indentation

File: test/e2e/features/cat-c1e/11-fail_journey_with multiple_faults.feature [hunk 2]
```diff
@@ -0,0 +1,57 @@
+@catc1e
+Feature: Full end to end journey for Category c1e
```
Comment (jono-basford-bjss) [inline test/e2e/features/cat-c1e/11-fail_journey_with multiple_faults.feature:2]: Rename Feature File

File: test/e2e/features/cat-c1e/99-fulljourney.feature [hunk 3]
```diff
@@ -0,0 +1,57 @@
+@catc1e
+Feature: Full end to end journey for Category c1e
```
Comment (jono-basford-bjss) [inline test/e2e/features/cat-c1e/99-fulljourney.feature]: These are duplicated file?

File: test/e2e/step-definitions/waitingroom-steps.ts [hunk 4]
```diff
@@ -18,7 +18,7 @@ Before({ tags: '@catc' }, () => {
   this.testCategory = 'c';
 });
 
-Before({ tags: '@catc1' }, () => {
+Before({ tags: '@catc1e' }, () => {
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroom-steps.ts:22]: We need this and we already have this tag

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 5]
```diff
@@ -59,6 +59,9 @@ When('I fail the eye sight test', () => {
 });
 
 When('I complete the waiting room to car page with the following vehicle checks', function (table) {
+  console.log('table', table);
+  console.log('--------------------');
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:62]: remove

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 6]
```diff
@@ -69,6 +72,7 @@ const completeWaitingRoomPage = (questionResult, manualTransmission: boolean, te
   } else if (this.testCategory === 'c' || this.testCategory === 'c1') {
     multiShowAndTell(UI_TEST_DATA.testData.c, questionResult);
   } else if (this.testCategory === 'ce') {
+    console.log('UI TEST DATA ce: ', UI_TEST_DATA.testData.ce);
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:74]: remove

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 7]
```diff
@@ -59,6 +59,9 @@ When('I fail the eye sight test', () => {
 });
 
 When('I complete the waiting room to car page with the following vehicle checks', function (table) {
+  console.log('table', table);
+  console.log('--------------------');
+  console.log('table raw', table.raw()[1]);
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:63]: remove

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 8]
```diff
@@ -105,8 +109,13 @@ const standardUserJourney = (withDriverFault: boolean, manualTransmission: boole
 
 const showMeQuestions = (questions, questionResult) => {
   const showMeQuestionsArray = [questions, questionResult];
+  console.log('SHOWMEQUESTIONSARRAY: ', showMeQuestionsArray);
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:111]: remove

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 9]
```diff
@@ -105,8 +109,13 @@ const standardUserJourney = (withDriverFault: boolean, manualTransmission: boole
...
   const elements = element.all(by.id('vehicle-checks-question-selector'));
   elements.each((element, index) => {
+    console.log('------------------------------------------------------------');
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:114]: remove

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 10]
```diff
@@ -105,8 +109,13 @@ const standardUserJourney = (withDriverFault: boolean, manualTransmission: boole
...
+    console.log('element', element);
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:115]: remove

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 11]
```diff
@@ -105,8 +109,13 @@ const standardUserJourney = (withDriverFault: boolean, manualTransmission: boole
...
+    console.log('------------------------------------------------------------');
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:116]: remove

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 12]
```diff
@@ -105,8 +109,13 @@ const standardUserJourney = (withDriverFault: boolean, manualTransmission: boole
...
+    console.log('index', index);
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:117]: remove

File: test/test_data/ui_test_data.ts [hunk 13]
```diff
@@ -3,6 +3,6 @@ export const UI_TEST_DATA = {
     be: ['S01 - Direction indicators', 'S02 - Doors secure', 'S03 - Horn', 'T01 - Brakes',
       'T02 - Safety factors while loading'],
     c: ['CD1 - All doors secure', 'CD2 - Air leaks', 'CD10 - Air pressure', 'CD7 - Sufficient tread', 'CD9 - Body safety'],
-    ce: ['E2 - Change tacho', 'E3 - Loading mechanism'],
+    ce: ['S1 - Change tacho', 'T2 - Loading mechanism'],
```
Comment (jono-basford-bjss) [inline test/test_data/ui_test_data.ts]: this is incorrect my pr will fix this

**Review by jono-basford-bjss — CHANGES_REQUESTED** [review]

File: test/e2e/features/cat-c1e/11-fail_journey_with multiple_faults.feature (hunk 1 above)
Comment (JCarterDVSA) [reply test/e2e/features/cat-c1e/11-fail_journey_with multiple_faults.feature:15]: done

File: test/e2e/features/cat-c1e/11-fail_journey_with multiple_faults.feature (hunk 2 above)
Comment (JCarterDVSA) [reply test/e2e/features/cat-c1e/11-fail_journey_with multiple_faults.feature:2]: done

File: test/e2e/step-definitions/waitingroom-steps.ts (hunk 4 above)
Comment (JCarterDVSA) [reply test/e2e/step-definitions/waitingroom-steps.ts:22]: done

File: test/e2e/step-definitions/waitingroomtocar-steps.ts (hunk 5 above)
Comment (JCarterDVSA) [reply test/e2e/step-definitions/waitingroomtocar-steps.ts:62]: done

File: test/e2e/step-definitions/waitingroomtocar-steps.ts (hunk 9 above)
Comment (JCarterDVSA) [reply test/e2e/step-definitions/waitingroomtocar-steps.ts:114]: done

File: test/e2e/step-definitions/waitingroomtocar-steps.ts (hunk 10 above)
Comment (JCarterDVSA) [reply test/e2e/step-definitions/waitingroomtocar-steps.ts:115]: done

File: test/e2e/step-definitions/waitingroomtocar-steps.ts (hunk 11 above)
Comment (JCarterDVSA) [reply test/e2e/step-definitions/waitingroomtocar-steps.ts:116]: done

File: test/e2e/step-definitions/waitingroomtocar-steps.ts (hunk 12 above)
Comment (JCarterDVSA) [reply test/e2e/step-definitions/waitingroomtocar-steps.ts:117]: done

**Pull request closed without merging**